src/jvm/main/org/jetbrains/kotlinx/lincheck/Options.kt

```diff
@@ -112,15 +111,6 @@ abstract class Options<OPT : Options<OPT, CTEST>, CTEST : CTestConfiguration> {
         this.verifier = verifier
     }
 
-    /**
-     * Require correctness check of test instance state equivalency relation defined by the user.
-     * It checks whether two new instances of a test class are equal.
-     * If the check fails [[IllegalStateException]] is thrown.
-     */
-    fun requireStateEquivalenceImplCheck(require: Boolean): OPT = applyAndCast {
```

Comment:
Please do not remove it, deprecate instead

Reply:
Done